conf/prod.routes

```diff
@@ -1,5 +1,5 @@
 # Add all the application routes to the app.routes file
-->         /income-tax-gift-aid        app.Routes
-->         /                           health.Routes
+->         /income-tax           app.Routes
```

Comment:
sorry the url wasn't in the ticket - think this is fine to be what it was /income-tax-gift-aid